packages/morph/builds/module.js

```diff
@@ -1,5 +1,5 @@
-import morphPlugin, { morph } from '../src/index.js'
```

Comment:
I think these are two separate exports and should remain that way.

But now I see that this would mean this plugin diverges from the rest of the named plugin exports.

Hmmmm.... the "morphPlugin" variable is actually the plugin, and "morph" is the morphing function, not the full plugin.

So this change would cause a breakage for other people consuming { morph }

It's not documented so I'm not sure how much of a breaking change it would be, but it at least will probably break something internally? or with Livewire?

Reply:
Just found at least one usage of this. [Jest tests are using this export](https://github.com/alpinejs/alpine/blob/main/tests/jest/morph/external.spec.js#L1).

Are this tests still used? They are not running on CI, and currently are failing.